packages/common-utils/src/renderChartConfig.ts

```diff
@@ -816,36 +816,80 @@ function translateMetricChartConfig(
       whereLanguage: 'sql',
     };
   } else if (metricType === MetricsDataType.Sum && metricName) {
+    const time_bucket_col = '`__hdx_time_bucket2`';
+    const value_high_col = '`__hdx_value_high`';
+    const value_high_prev_col = '`__hdx_value_high_prev`';
+    const time_expr = timeBucketExpr({
+      interval: chartConfig.granularity || 'auto',
+      timestampValueExpression:
+        chartConfig.timestampValueExpression || 'TimeUnix',
+      dateRange: chartConfig.dateRange,
+      alias: '__hdx_time_bucket2',
```

Comment:
nit: can be `time_bucket_col`

Reply:
`timeBucketCol` contained quote characters, which can't be passed here. I moved those quote char into the SQL block, leaving this name the same. Maybe it's clearer?